plugins/database/mysql/connection_producer.go

```diff
@@ -108,20 +107,6 @@ func (c *mySQLConnectionProducer) Init(ctx context.Context, conf map[string]inte
 		mysql.RegisterTLSConfig(c.tlsConfigName, tlsConfig)
 	}
 
-	// Set initialized to true at this point since all fields are set,
```

Comment:
Why are we remove this code block from `Init`? Since this func deals with the connection logic, it makes sense to keep this verification within `mySQLConnectionProducer.Init` instead of moving it into `MySQL.Initialize`.

Reply:
Because it sets the `Initialized` variable when it hasn't finished initializing. In working on the next DB I realized I could move this back and shuffle some other logic around to make that happy without changing this.

Reply:
Can we restore this deleted godoc line?

Reply:
Done. I missed adding it back in previously =/